.github/workflows/tracker-integration.yaml

```diff
@@ -71,3 +71,8 @@ jobs:
 
           kubectl wait --for=condition=complete --timeout=300s job/depscloud-indexer
           kubectl logs -l job-name=depscloud-indexer
+
+      - name: Run integration tests
+        run: |
+          cd tracker
+          make test-integration
```

Comment:
I think, before running tests, doing a `kubectl port-forward` to connect to postgreSql running inside Minikube from the Github action runner will work. Shall look at it.